src/storage/src/source/resumption.rs

```diff
@@ -77,6 +77,9 @@ where
         // TODO: determine what interval we want here.
         let mut interval = tokio::time::interval(std::time::Duration::from_secs(10));
         interval.set_missed_tick_behavior(tokio::time::MissedTickBehavior::Skip);
+        // Clear the first instantaneous tick
+        // See <https://docs.rs/tokio/latest/tokio/time/struct.Interval.html#method.tick>
+        interval.tick().await;
```

Comment:
Should we call `interval.reset();` instead?

Reply:
i find that less clear from the docs!